Sources/Vapor/Application.swift

```diff
@@ -111,7 +116,7 @@ public final class Application: Sendable {
 
     public init(
         _ environment: Environment = .development,
-        _ eventLoopGroupProvider: EventLoopGroupProvider = .createNew
+        _ eventLoopGroupProvider: EventLoopGroupProvider = .shared(NIOSingletons.posixEventLoopGroup)
```

Comment:
Since you declared a `.singleton`, shouldn't this use it so there's less to refactor if the default should be a different singleton in the future?
```suggestion
        _ eventLoopGroupProvider: EventLoopGroupProvider = .singleton
```